addons/web/static/src/search/control_panel/control_panel.js

```diff
@@ -54,6 +54,7 @@ export class ControlPanel extends Component {
     static props = {
         display: { type: Object, optional: true },
         slots: { type: Object, optional: true },
+        hideGlobalFilters: { type: Boolean, optional: true },
```

Comment:
Remove this. This should not impact `web` module.
It should only impact `spreadsheet_dashboard`